episodes/01-intro.md

```diff
@@ -26,52 +26,39 @@ The most widely used way to interact with personal computers is called a
-This is where we take advantage of the Unix shell.
-The Unix shell is both a **command-line interface** (CLI) and a scripting language,
-allowing such repetitive tasks to be done automatically and fast.
-With the proper commands, the shell can repeat tasks with or without some modification
-as many times as we want.
-Using the shell, the task in the literature example can be accomplished in seconds.
-
-### The Shell
-
-The shell is a program where users can type commands.
-With the shell, it's possible to invoke complicated programs like climate modeling software
-or simple commands that create an empty directory with only one line of code.
-The most popular Unix shell is Bash (the Bourne Again SHell ---
-so-called because it's derived from a shell written by Stephen Bourne).
-Bash is the default shell on most modern implementations of Unix and in most packages that provide
-Unix-like tools for Windows.
-Note that 'Git Bash' is a piece of software that enables Windows users to use a Bash like interface
-when interacting with Git.
+
+Using a CLI, you can write a series of commands that does this consistently and nearly instantly.
+
+The shell can be used for simple tasks like creating an empty folder
+and for launching (even complex) programs with a single command.
+In fact, some tools and resources such as cloud computing systems
+usually require users to be familiar with the shell.
+Shell commands can be combined together and saved into reproducible *scripts*
+that handle large volumes of data automatically.
 
 Using the shell will take some effort and some time to learn.
-While a GUI presents you with choices to select, CLI choices are not automatically presented to you,
-so you must learn a few commands like new vocabulary in a language you're studying.
-However, unlike a spoken language, a small number of "words" (i.e. commands) gets you a long way,
-and we'll cover those essential few today.
-
-The grammar of a shell allows you to combine existing tools into powerful
-pipelines and handle large volumes of data automatically. Sequences of
-commands can be written into a *script*, improving the reproducibility of
-workflows.
-
-In addition, the command line is often the easiest way to interact with remote machines
-and supercomputers.
-Familiarity with the shell is near essential to run a variety of specialized tools and resources
-including high-performance computing systems.
-As clusters and cloud computing systems become more popular for scientific data crunching,
-being able to interact with the shell is becoming a necessary skill.
-We can build on the command-line skills covered here
-to tackle a wide range of scientific questions and computational challenges.
-
-Let's get started.
+While a GUI presents you with choices to select, CLI choices are not automatically presented to you. It can be daunting at first, but once you've come familliar with this different style
```

Comment:
```suggestion
While a GUI presents you with choices to select, CLI choices are not automatically presented to you.
It can be daunting at first, but once you've come familliar with this different style
```
Break long line.